app/logic/emailHandler.py

```diff
@@ -66,13 +70,18 @@ def process_data(self):
         if 'slCourseId' in self.raw_form_data:
             self.sl_course_id = self.raw_form_data['slCourseId']
 
-    def fetch_event_programs(self, program_id):
-        """ Fetches all the programs of a particular event """
-        # Non-student-led programs have "Unknown" as their id
-        if program_id == 'Unknown' or program_id is None:
-            return self.event.program
-        else:
-            return Program.get_by_id(program_id)
+
+    def getReplyInfo(self):
+        programObject = Program.get_or_none(Program.programName == self.sender_username)
+        userobj = User.get_or_none(User.username == self.sender_username)
+        if programObject:
+            programEmail = programObject.contactEmail
+            return (programObject.programName, programEmail, programEmail)
+        elif self.sender_username.upper() == "CELTS":
+            return ("CELTS", "celts@berea.edu", "celts@berea.edu")
+        elif userobj:
+            return (f"{userobj.fullName}", userobj.email, userobj.email)
+        return (None, None, None) # If the sender is not a program or user, use default values.
```

Comment:
I understand what you mean but maybe change the wording of the comment. A "program" cant be a sender.

Reply:
I made it better.